<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HStimulusInfoForm</class>
 <widget class="QWidget" name="HStimulusInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>54</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelPosition">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineeditFileBase">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>no stimulus selected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbSelectFile">
     <property name="toolTip">
      <string>Select stimulus file</string>
     </property>
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sliderVolume">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Set volume level (ignored for images and colors)</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkboxLoop">
     <property name="toolTip">
      <string>Repeat stimulus file in a loop? (ignored for images and colors)</string>
     </property>
     <property name="text">
      <string>Loop?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbUseBackgroundColor">
     <property name="text">
      <string>Background color only</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
